test/remote/gateways/remote_orbital_test.rb

```diff
@@ -829,6 +830,96 @@ def test_successful_verify
     assert_equal 'No reason to decline', response.message
   end
 
+  def test_successful_store
+    response = @tpv_orbital_gateway.store(@mastercard_card_tpv, @options)
+    assert_success response
+    assert_false response.params['safetech_token'].blank?
+  end
+
+  def test_successful_purchase_stored_token
+    store = @tpv_orbital_gateway.store(@credit_card, @options)
+    assert_success store
+    response = @tpv_orbital_gateway.purchase(@amount, store.authorization, @options.merge(card_brand: 'VI', token_txn_type: 'UT'))
```

Comment:
it would be helpful to include more descriptive arguments in tests to provide guidance for future developers or even to us. Why?, same as above, it is not clear what "UT" means and the effects that value has.

Reply:
yes, I will add some comment to clarify